packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -311,6 +340,32 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     return renderedItems;
   }
 
+  private _renderSectionItem(item: IContextualMenuItem, index: number, hasCheckmarks: boolean, hasIcons: boolean) {
+    const section = item.sectionProps;
+    if (!section) {
+      return;
+    }
+
+    let headerItem = section.title ? this._renderHeaderMenuItem(section.title, index, hasCheckmarks, hasIcons) : undefined;
```

Comment:
Several instances around here could just be `&&` statements instead of ternaries that end with `: undefined`